example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,100 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  tracks: Membrane.Track[];
+  metadata: Membrane.Metadata;
+  type: Membrane.ParticipantType;
+};
+
+export const RoomParticipant = ({
+  tracks,
+  metadata,
+  type,
+}: RoomParticipantProps) => {
+  const videoTrack = tracks.find((t) => t.type === 'Video');
+  const audioTrack = tracks.find((t) => t.type === 'Audio');
+
+  const participantHasVideo = () => {
+    if (videoTrack !== undefined) {
```

Comment:
Why not `if(videoTrack)`? In js values might be `undefined`, `null` and other weird values. We don't want to think about them and `if(videoTrack)` should cover them all.

Reply:
Fixed.